stage0/app.go

```diff
@@ -357,7 +357,12 @@ func RmApp(dir string, uuid *types.UUID, usesOverlay bool, appName *types.ACName
 		}
 
 		if err := callEntrypoint(dir, appStopEntrypoint, args); err != nil {
-			return err
+			status, err := common.GetExitStatus(err)
+			// ignore nonexistent units failing to stop. Exit status 5
+			// comes from systemctl and means the unit doesn't exist
+			if err != nil || status != 5 {
```

Comment:
hmm afais `GetExitStatus` doesn't maintain the invariant `status != 0 && err != nil`, hence this can return `nil`.